src/apps/viewer/index.ts

```diff
@@ -251,6 +258,22 @@ export class Viewer {
         }));
     }
 
+    loadAfdb(afdb: string) {
```

Comment:
Maybe be explicit here: ``loadAlphaFoldDb``?

Reply:
sounds better, done